### PR TITLE
Add pilot CRDs in istio

### PR DESCRIPTION
**Pull request by andraxylia**
Fixes istio/pilot#1328 in istio. We need a corresponding change in istio/pilot repo.

### CODE REVIEW

**Comment by kyessenov** [comment]
cc @mandarjog; it would be good to label all CRDs with `istio:`

File: install/kubernetes/templates/istio-ingress.yaml.tmpl [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ################################
-# Istio ingress controller
+# Istio-ingress
```
Comment (rkpagadala) [inline install/kubernetes/templates/istio-ingress.yaml.tmpl]: sugestion: please change to "Istio-ingress" to "Istio ingress"

**Comment by linsun** [comment]
/lgmt.  Would you also need to make change on pilot to remove these crds from code?

File: install/kubernetes/templates/istio-pilot.yaml.tmpl [hunk 2]
```diff
@@ -1,3 +1,49 @@
+################################
+# Pilot
+################################
+# Pilot CRDs
```
Comment (ldemailly) [inline install/kubernetes/templates/istio-pilot.yaml.tmpl]: how do we know there are no typos / won't be typos in the future ? does some test fail ? some error in logs ?

**Comment by andraxylia** [comment]
@ldemailly @rkpagadala ptal and approve again, approval is lost after I changed the comment.

File: install/kubernetes/templates/istio-pilot.yaml.tmpl (hunk 2 above)
Comment (andraxylia) [reply install/kubernetes/templates/istio-pilot.yaml.tmpl]: Routing rules would fail @kyessenov may have more details.

**Comment by mandarjog** [comment]
@andraxylia and the crds should have
Labels:
   Istio: proxy

That we can find all istio crd with '-l istio'

**Comment by ldemailly** [comment]
@mandarjog proxy ? or pilot ? or sidecar ? I guess it's not absolutely necessary and could have been added later (the goal here was for `kubectl delete -f istio.yaml` to delete everything which it would do without labels)

**Comment by andraxylia** [comment]
/tickle

feel free to add labels in another PR, there is no need to block this one by applying do not merge.

**Pull request merged**